### PR TITLE
test(dpp): double test in identity validation

### PR DESCRIPTION
**Pull request by antondash**
## Issue being fixed or feature implemented

Identity validation tests having some absolutely identical tests

## What was done?

Fix the test to match its description

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js [hunk 1]
```diff
@@ -369,9 +369,9 @@ describe('validateIdentityFactory', () => {
   });
 
   it('should return valid result if an identity model is valid', () => {
-    const result = validateIdentity(rawIdentity);
+    const result = validateIdentity(identity);
```
Comment (shumkov) [inline packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js:372]: `validateIdentity` expects rawIdentity but not an instance of Identity.

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js (hunk 1 above)
Comment (antondash) [reply packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js:372]: It's literally what the test name implies: should return valid result if an **identity model** is valid. There's another test for the raw identity right above that one

Comment (shumkov) [reply packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js:372]: Discussed it verbally: this function doesn't accept model anymore only raw and it seems this test is outdated and just needs to be removed.

**Review by shumkov — DISMISSED** [review]
👍

**Review by shumkov — APPROVED** [review]

**Pull request merged**